example/simple/coreapi-3dsnewflow/views/index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
```

Comment:
Rename this example folder name to `coreapi-card-3ds`. We don't need to explicitly say `new flow`.